components/etl/intro/Intro.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC (http://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import * as React from 'react';
+import { Row, Col, Container } from 'react-bootstrap';
+import Image from 'next-image-export-optimizer';
+
+import { prefix } from '../../../utils/prefix';
+import styles from './Intro.module.css';
+
+export default function Intro() {
+
+    return (
+        <>
+            <Col xs={12}>
+                <Container>
+                    <Row className='pageContentRow integration'>
+                        <Col xs={12} lg={5} className={styles.introText}>
+                            <div>
+
+                                <p className={styles.introText2}>
+                                    Extracting, transforming, and loading data often involves dealing with intricate, time-consuming processes, leading to high costs and operational inefficiencies.
+                                </p>
+
+                                <p className={styles.introText3}>
+                                    Ballerina simplifies ETL with its programmable approach. It excels in complex data operations, integrations, and deployment flexibility, ensuring a scalable and adaptable ETL experience.
+                                </p>
+
+                                <p className={styles.dVersion}>
+                                    <a href={`${prefix}/downloads/`}
+                                        className={styles.cDownload}>
+                                        Download Ballerina
+                                    </a>
+                                </p>
+                            </div>
+                        </Col>
+                        <Col xs={12} lg={7} className={styles.introImg}>
+                            {/* <Image src='/images/maxresdefault.jpg' alt='Integration' width={1280} height={780} /> */}
+                            <img src={`${prefix}/images/ballerina-etl-sm-banner.png`} alt="AI" />
```

Comment:
```suggestion
                            <img src={`${prefix}/images/etl-sm-banner.png`} alt="AI" />
```
ballerina-etl-sm-banner.png used only when you share the link on social media. Add the deleted etl-sm-banner.png again.

Reply:
updated in [commit](https://github.com/ballerina-platform/ballerina-dev-website/pull/8758/commits/230a1ef278dfea5c1c7e7389651b61573c68a22b).